### PR TITLE
Add zig language run support

### PR DESCRIPTION
**Pull request by forinda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 
-Run code snippet or code file for multiple languages: **C, C++, Java, JavaScript, PHP, Python, Perl, Perl 6, Ruby, Go, Lua, Groovy, PowerShell, BAT/CMD, BASH/SH, F# Script, F# (.NET Core), C# Script, C# (.NET Core), VBScript, TypeScript, CoffeeScript, Scala, Swift, Julia, Crystal, OCaml Script, R, AppleScript, Elixir, Visual Basic .NET, Clojure, Haxe, Objective-C, Rust, Racket, Scheme, AutoHotkey, AutoIt, Kotlin, Dart, Free Pascal, Haskell, Nim, D, Lisp, Kit, V, SCSS, Sass, CUDA, Less, Fortran, Ring, Standard ML**, and custom command
```
Comment (formulahendry) [inline README.md:5]: Just append to the end.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "code-runner",
   "displayName": "Code Runner",
-  "description": "Run C, C++, Java, JS, PHP, Python, Perl, Ruby, Go, Lua, Groovy, PowerShell, CMD, BASH, F#, C#, VBScript, TypeScript, CoffeeScript, Scala, Swift, Julia, Crystal, OCaml, R, AppleScript, Elixir, VB.NET, Clojure, Haxe, Obj-C, Rust, Racket, Scheme, AutoHotkey, AutoIt, Kotlin, Dart, Pascal, Haskell, Nim, D, Lisp, Kit, V, SCSS, Sass, CUDA, Less, Fortran, Ring, Standard ML",
```
Comment (formulahendry) [inline package.json:4]: Same here: Just append to the end.

File: package.json [hunk 3]
```diff
@@ -135,6 +135,7 @@
             "javascript": "node",
             "java": "cd $dir && javac $fileName && java $fileNameWithoutExt",
             "c": "cd $dir && gcc $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
+            "zig": "zig run $fileName",
```
Comment (formulahendry) [inline package.json:138]: No `$fileName` needed

File: package.json [hunk 4]
```diff
@@ -196,6 +197,7 @@
             ".jl": "julia",
             ".cr": "crystal",
             ".ml": "ocaml",
+            ".zig": "zig run $fileName",
```
Comment (formulahendry) [inline package.json:200]: Same here: No `$fileName` needed

**Review by formulahendry — CHANGES_REQUESTED** [review]
Others look good to me~

**Comment by forinda** [comment]
Let me rework on it

File: package.json (hunk 2 above)
Comment (forinda) [reply package.json:4]: Ok

File: package.json (hunk 4 above)
Comment (forinda) [reply package.json]: Resolved the `$fileName`

File: package.json (hunk 3 above)
Comment (forinda) [reply package.json]: Removed the `$fileName`

File: package.json (hunk 2 above)
Comment (formulahendry) [reply package.json:4]: Please also update this one as well, 😃

**Review by formulahendry — CHANGES_REQUESTED** [review]
Just leave one comment~

**Comment by forinda** [comment]
> Just leave one comment~

Done

**Review by formulahendry — CHANGES_REQUESTED** [review]
I still not see the updates:
image

BTW, `zig` should be `Zig`?

**Comment by forinda** [comment]
> I still not see the updates:
> image
> 
> BTW, `zig` should be `Zig`?

 In the latest commit do you mean zig in the readme to be capitalized

**Review by forinda — COMMENTED** [review]
Capitalize zig

**Comment by formulahendry** [comment]
> > I still not see the updates:
> > image
> > BTW, `zig` should be `Zig`?
> 
> In the latest commit do you mean zig in the readme to be capitalized

@forinda I mean this line: https://github.com/forinda/vscode-code-runner/blob/5c5534480c8576d56613007e224a6ee1a665e7e9/package.json#L4_

**Comment by forinda** [comment]
> > > I still not see the updates:
> > > image
> > > BTW, `zig` should be `Zig`?
> > 
> > 
> > In the latest commit do you mean zig in the readme to be capitalized
> 
> @forinda I mean this line: https://github.com/forinda/vscode-code-runner/blob/5c5534480c8576d56613007e224a6ee1a665e7e9/package.json#L4_

Updated

**Review by formulahendry — APPROVED** [review]

**Pull request merged**